web/src/app/admin/connectors/google-drive/page.tsx

```diff
@@ -317,12 +329,21 @@ const Main = () => {
     );
   }
 
+  if (iscurrentUserError || !credentialsData) {
+    return (
+      <div className="mx-auto">
+        <div className="text-red-500">
+          Error loading user profile. Contact an administrator.
+        </div>
+      </div>
+    );
+  }
   const googleDrivePublicCredential:
     | Credential<GoogleDriveCredentialJson>
     | undefined = credentialsData.find(
     (credential) =>
       credential.credential_json?.google_drive_tokens &&
-      credential.user_id === null
+      credential.user_id === currentUserData?.id
```

Comment:
Another way of doing this that imo better replicates the existing logic is to change this check to `credential.credential_json?.google_drive_tokens && credential.is_admin`. 

For now, we generally assume that there should/could only be one live admin credential. This keeps the logic here simpler for now, and enables multiple admins to see / modify each others connectors.

Let me know if that makes sense / if you have questions about that!

Reply:
Totally agree, better to keep the logic simple atm